submissions/ScreenLimits/README.md

```diff
@@ -0,0 +1,14 @@
+# Screen Time Manager

```

Comment:
<img width="388" alt="image" src="https://github.com/user-attachments/assets/8bb0800c-1688-4d7a-bb6a-31f1544cc013" />

How does it calculate the time spent? I've left my browser open for more than two minutes including on sites in my blocklist but it never seems to elapse?
It'll be nice to also include instructions on how to use it